### PR TITLE
Add Swagger

### PR DESCRIPTION
**Pull request by alanlamaswolox**
## Summary
Added Swagger to API

## Trello
[link](https://trello.com/c/bTcrbBs9/40-swagger)

### CODE REVIEW

**Review by Gianni-Nicolas — DISMISSED** [review]

File: src/main/java/wolox/training/controllers/BookController.java [hunk 1]
```diff
@@ -34,7 +40,15 @@ public Book findByAuthor(@PathVariable String author) throws BookNotFoundExcepti
      * @throws BookNotFoundException (If we can't find a book by this id)
      */
     @PutMapping("/{id}")
-    public Book updateBook(@PathVariable Long id, @RequestBody Book book) throws BookNotFoundException{
+    @ApiOperation(value = "Giving an Id and a Book, it updates that book", response = Book.class)
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = "Successfuly updated book"),
+        @ApiResponse(code = 404, message = "Book Not Found")
```
Comment (AleFalcon) [inline src/main/java/wolox/training/controllers/BookController.java]: You can add more errors, for example, that you don't access, error internal server, method not allowed, etc.

File: src/main/java/wolox/training/models/Book.java [hunk 2]
```diff
@@ -27,6 +31,7 @@
     @Column(nullable = false)
     private String image;
 
+    @ApiModelProperty(notes = "Book's title")
```
Comment (AleFalcon) [inline src/main/java/wolox/training/models/Book.java]: can you are add this format "@ApiModelProperty(notes = "Book's tittle", required = true)", so you indicate that this field is required

File: src/main/java/wolox/training/models/Book.java [hunk 3]
```diff
@@ -27,6 +31,7 @@
...
     @Column(nullable = false)
```
Comment (AleFalcon) [inline src/main/java/wolox/training/models/Book.java:38]: Add ApiModel. same comment as before

**Review by AleFalcon — COMMENTED** [review]
It's okay, that some comments

**Review by AleFalcon — DISMISSED** [review]
Okay!

**Review by AleFalcon — APPROVED** [review]

**Review by Gianni-Nicolas — APPROVED** [review]

**Pull request merged**